src/main/java/core/gui/model/ArenaStatistikTableModel.java

```diff
@@ -274,7 +274,7 @@ private void initData() {
 				colIndex++;
 
 			    //Ergebnis
-			    m_clData[i][colIndex] = new ColorLabelEntry(StringUtils.getResultString(match.getHeimTore(), match.getGastTore()),
+			    m_clData[i][colIndex] = new ColorLabelEntry(StringUtils.getResultString(match.getHeimTore(), match.getGastTore(), ""),
```

Comment:
i changed the signature of the getResultString method to get an idea of ​​where it is used. We can now decide whether we want additional text in the corresponding displays or not. In some displays it would not be easy to reach MatchesKurzInfo.